Update teams-powershell-release-notes.md (#6958)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-powershell-release-notes.md

| Date | Version | Updates |
|------- | -------------------- | ------------------------------ |

> | April 2021 | [2.0.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.1.0-preview) | <li>Fixed formatting of existing cmdlets (Eg: Get-CsTeamsNetworkRoamingPolicy, Get-CsTeamsMeetingPolicy, Get-CsTeamsMessagingPolicy, etc.)</li><li>Updated parameter list of policy management cmdlets.</li>|

Reviewer: ```suggestion
| April 2021 | [2.0.1-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.1.0-preview) | <li>Fixed formatting of existing cmdlets (for example, Get-CsTeamsNetworkRoamingPolicy, Get-CsTeamsMeetingPolicy, Get-CsTeamsMessagingPolicy, and more).</li><li>Updated parameter list of policy management cmdlets.</li>|
```
